frontend/components/Course/CourseWrapper.tsx

```diff
@@ -116,7 +117,7 @@ const CourseWrapper = ({ render, ...props }: CourseProps) => {
                                             </p>
                                             {typeof Notification !==
                                                 "undefined" &&
-                                                Notification.permission ==
+                                                Notification.permission ===
```

Comment:
Did you mean to use `getPermissions` here?

Reply:
This should work: https://developer.mozilla.org/en-US/docs/Web/API/Notification/permission

Reply:
Ah, this was referring to the function "getPermissions()" defined below, but I'm fine with it being removed and inlined here